keps/sig-storage/20200120-skip-permission-change/README.md

```diff
@@ -111,6 +95,67 @@ type PodSecurityContext struct {
 
 - One of the risks is if user volume's permission was previously changed using old algorithm(which changes permission of top level directory first) and user opts in for `OnRootMismatch` `FSGroupChangePolicy` then we can't distinguish if the volume was previously only partially recursively chown'd.
 
+## Production Readiness Review Questionnaire
+
+### Feature enablement and rollback
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [ ] Feature gate
+    - Feature gate name: ConfigurableFSGroupPolicy
+    - Components depending on the feature gate: kubelet, kube-apiserver, kube-controller-manager
```

Comment:
Does kube-controller-manager depend on it?

Reply:
Strictly speaking no. But paranoid in me wants to ensure that something in kube-controller-manager does not cause the newly introduced field to be dropped.

Reply:
This is intended to help guide the operators when they enable or disable. If no change is required in the k-c-m config, it shouldn't be listed here.

Reply:
I dropped k-c-m from feature gate docs.